modules/cco-determine-mode-cli.adoc

```diff
@@ -0,0 +1,120 @@
+// Module included in the following assemblies:
+//
+// * updating/updating-cluster-cli.adoc
+// * authentication/managing_cloud_provider_credentials/about-cloud-credential-operator.adoc
+
+:_content-type: PROCEDURE
+
+ifeval::["{context}" == "updating-cluster-cli"]
+:cli:
+endif::[]
+ifeval::["{context}" == "about-cloud-credential-operator"]
+:about-cco:
+endif::[]
+
+[id="cco-determine-mode-cli_{context}"]
+= Determining the Cloud Credential Operator mode by using the CLI
+
+You can determine what mode the Cloud Credential Operator (CCO) is configured to use by using the CLI.
+
+[NOTE]
+====
+Only Amazon Web Services (AWS), global Microsoft Azure, and Google Cloud Platform (GCP) clusters support multiple CCO modes.
+====
+
+.Prerequisites
+
+* You have access to an {product-title} account with cluster administrator access.
+* You have installed the OpenShift CLI (`oc`).
+
+.Procedure
+
+. Log in to `oc` on the cluster as a user with the `cluster-admin` role.
+
+. To determine the mode that the CCO is configured to use, enter the following command:
++
+[source,terminal]
+----
+$ oc get cloudcredentials cluster \
+  -o=jsonpath={.spec.credentialsMode}
+----
++
+The following output values are possible, though not all are supported on all platforms:
++
+--
+* `''`: The CCO is operating in the default mode. In this configuration, the CCO operates in mint or passthrough mode, depending on the credentials provided during installation.
```

Comment:
I think this is correct but we should double check that `''` can never mean `"Manual"`.

Reply:
Per Slack discussion, probably:
* In [Default behavior](https://57250--docspreview.netlify.app/openshift-enterprise/latest/authentication/managing_cloud_provider_credentials/about-cloud-credential-operator.html#about-cloud-credential-operator-default_about-cloud-credential-operator), we say that by default (`''`), the CCO tries the creds it has for mint. If they won't mint, it tries them for passthrough. If they're insufficient for passthrough, "the installation fails."
* In the AliCloud basic (no customizations) [install](https://docs.openshift.com/container-platform/4.12/installing/installing_alibaba/installing-alibaba-default.html#installation-initializing_installing-alibaba-default), the single change the user must make to the `install-config.yaml` file is to set the `credentialsMode` parameter to `Manual`.

But it would be good to get QE to confirm this is the case :slightly_smiling_face: